docs/getting-started/installation-and-upgrade/installation-references/feature-flags.md

```diff
@@ -28,6 +28,7 @@ The following is a list of feature flags available in Rancher. If you've upgrade
 - `rke1-custom-node-cleanup`: Enables cleanup of deleted RKE1 custom nodes. We recommend that you keep this flag enabled, to prevent removed nodes from attempting to rejoin the cluster.
 - `rke2`: Enables provisioning RKE2 clusters. This flag is enabled by default.
 - `token-hashing`: Enables token hashing. Once enabled, existing tokens will be hashed and all new tokens will be hashed automatically with the SHA256 algorithm. Once a token is hashed it can't be undone. This flag can't be disabled after its enabled. See [API Tokens](../../../api/api-tokens.md#token-hashing) for more information.
+- `uiextension`: Allows UI extensions to load without authentication. The `noAuth` field in the CR must be set to `true` for this feature to work. This flag is enabled by default. Enabling or disabling the flag forces Rancher to restart. The first time this flag is set to `true`, it creates a CRD and enables the controllers and endpoints it needs for the feature to work. If set to `false`, it disables the previously mentioned controllers and endpoints. Setting `uiextension` to `false` has no effect on the CRD -- it will not create a CRD if it does not yet exist, nor will it delete the CRD if it already exists. 
```

Comment:
```suggestion
- `uiextension`: Allows UI extensions to load without authentication and is enabled by default. The `noAuth` field in the CR must be set to `true` for this feature to work. Enabling or disabling the flag forces Rancher to restart. The first time this flag is set to `true`, it creates a CRD and enables the controllers and endpoints it needs for the feature to work. If set to `false`, it disables the previously mentioned controllers and endpoints. Setting `uiextension` to `false` has no effect on the CRD -- it will not create a CRD if it does not yet exist, nor will it delete the CRD if it already exists. 
```

Reply:
Slight rewording to mention default enablement in first sentence for users.

Reply:
Thanks for the review. I tried to follow the structure of other flag descriptions on the page -- "[name]: Enables [feature]. This flag is [enabled/disabled] by default." I also wanted to come out and state that setting the flag to `false` doesn't affect the CRD. I realize the suggested wording is similar/the same as the release note, but that's why I made the changes I did.

Reply:
Makes sense, looks good to me!